index.html

```diff
@@ -8222,6 +8224,548 @@
+      <p>The <code>RTCQuicTransport.transport.onstatechange</code> EventHandler
+      can be used to determine whether an <code><a>RTCQuicTransport</a></code>
+      transitions to the desired role. When
+      <code><a>RTCQuicTransport</a>.transport.state</code> transitions to
+      <code>connected</code>, if <code><a>RTCQuicTransport</a>.transport.role</code>
+      is <code>controlled</code> then the role of the
+      <code><a>RTCQuicTransport</a></code> is <code>client</code>.
+      If <code><a>RTCQuicTransport</a>.transport.role</code>
+      is <code>controlling</code> then the role of the
+      <code><a>RTCQuicTransport</a></code> is <code>server</code>.</p>
+    </section>
+    <section id="rtcquictransportstate*">
+      <h3>enum RTCQuicTransportState</h3>
+      <p><dfn><code>RTCQuicTransportState</code></dfn> indicates the state of the QUIC
+      transport.</p>
+      <div>
+        <pre class="idl">enum RTCQuicTransportState {
+    "new",
+    "connecting",
+    "connected",
+    "closed",
+    "failed"
+};</pre>
+        <table data-link-for="RTCQuicTransportState" data-dfn-for="RTCQuicTransportState"
+        class="simple">
+          <tbody>
+            <tr>
+              <th colspan="2">Enumeration description</th>
+            </tr>
+            <tr>
+              <td><dfn><code id="idl-def-RTCQuicTransportState.new">new</code></dfn></td>
+              <td>
+                <p>The <code><a>RTCQuicTransport</a></code> object has been created and
+                has not started negotiating yet.</p>
+              </td>
+            </tr>
+            <tr>
+              <td><dfn><code id=
+              "idl-def-RTCQuicTransportState.connecting">connecting</code></dfn></td>
+              <td>
+                <p>QUIC is in the process of negotiating a secure connection and
+                verifying the remote fingerprint. Once a secure connection is negotiated
+                (but prior to verification of the remote fingerprint, enabled by calling
+                <code>start()</code>), incoming data can flow through.</p>
+              </td>
+            </tr>
+            <tr>
+              <td><dfn><code id=
+              "idl-def-RTCQuicTransportState.connected">connected</code></dfn></td>
+              <td>
+                <p>QUIC has completed negotiation of a secure connection and verified the
+                remote fingerprint. Outgoing data and media can now flow through.</p>
+              </td>
+            </tr>
+            <tr>
+              <td><dfn><code id=
+              "idl-def-RTCQuicTransportState.closed">closed</code></dfn></td>
+              <td>
+                <p>The QUIC connection has been closed intentionally via a call to
+                <code>stop()</code> or receipt of a close_notify alert. Calling
+                <code>transport.stop()</code> will also result in a transition to the
+                <code>closed</code> state.</p>
```

Comment:
[QUIC Section 10.2](https://tools.ietf.org/html/draft-ietf-quic-transport-04#section-10.2) defines the state machine, which includes a "closed" state.